cl_sii/dte/constants.py

```diff
@@ -6,6 +6,187 @@
 
 """
 import enum
+from datetime import date
+from typing import FrozenSet
+
+
+@enum.unique
+class TipoDocumentoEnum(enum.IntEnum):
```

Comment:
Please see `RcvTipoDocto` https://github.com/fyntex/lib-cl-sii-python/blob/6d84a11b235f9302516a7972347a33178b6a9373/cl_sii/rcv/constants.py#L101
At first sight it seems to be pretty similar so a merge/refactor might be in order.

Also, since by "Documento" here we mean something that might not be a DTE necessarily, I think it should in another module. Perhaps in `cls_sii.base.constants` ? I'm not sure where.